openquake/server/views.py

```diff
@@ -726,7 +702,15 @@ def aristotle_validate(request):
                          "invalid_inputs": invalid_inputs}
         return HttpResponse(content=json.dumps(response_data),
                             content_type=JSON, status=400)
-    return lon, lat, dep, mag, rake, dip, strike, maximum_distance, trt
+    # FIXME: I tried using a list comprehension like
+    # return [locals()[fieldname] for fieldname in field_validation]
+    # but it did not work
+    return (locals()['lon'], locals()['lat'], locals()['dep'], locals()['mag'],
+            locals()['rake'], locals()['dip'], locals()['strike'],
+            locals()['maximum_distance'], locals()['trt'],
+            locals()['truncation_level'],
+            locals()['number_of_ground_motion_fields'],
+            locals()['asset_hazard_distance'])
 
```

Comment:
Why using locals()? It is horrible. You can always define a class AristotleParameters and use that.